.docker/docker-qgis-test.sh

```diff
@@ -10,9 +10,13 @@ set -e
 ##################################
 
 if [ ${HANA_TESTS_ENABLED:-"OFF"} == "ON" ] ; then
-  echo "::group::hana"
```

Comment:
this is used to have nicer output on the workflow.
any reason to remove this?

Reply:
I did it by analogue with other database initialization sections (PostgreSQL, Oracle) where it was removed.

Reply:
Can you replace them? They are useful in the log.
Otherwise LGTM

Reply:
Sure, done.